publish-pr-preview/entrypoint.sh

```diff
@@ -92,8 +96,9 @@ EOT
         run_danger
         exit 1
       else
-        echo "registry=https://registry.npmjs.org/" >> .npmrc
-        echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" >> .npmrc
+        echo 'publishin'
+        # echo "registry=https://registry.npmjs.org/" >> .npmrc
+        # echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" >> .npmrc
```

Comment:
This is done away with in #47 and so commented out here. We should merge that one first and then rebase this on that later.